community/cypher/frontend-3.0/src/main/scala/org/neo4j/cypher/internal/frontend/v3_0/symbols/TypeSpec.scala

```diff
@@ -36,6 +36,7 @@ object TypeSpec {
     CTNumber,
     CTPath,
     CTRelationship,
+    CTGeometry,
```

Comment:
Unfortunately it's not separate, as the types get exposed in the type system. I.e. you see error messages like "expected Geometry" rather than the more obvious "expected Point". And I agree we only need one right now - I just suggest it's CTPoint, since that's what we're working with. Adding more later is then no conflict. If you wanted, you could have CTPoint->CTGeometry->CTAny, but it's not necessary.